### PR TITLE
Global css position

### PR DESCRIPTION
**Pull request by shivani170**
# Description

Fixes https://github.com/devtron-labs/devtron/issues/2036

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# Checklist:

* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/EnvironmentOverride/DeploymentTemplateOverride.tsx [hunk 1]
```diff
@@ -304,8 +304,8 @@ function DeploymentTemplateOverrideForm({
                     type="deployment template"
                 />
                 <div className="form__row">
-                    <div className="m-b-4 form__label">Chart type</div>
-                    <div className="text__subtitle">{chartName}</div>
+                    <div className="mb-4-imp form__label">Chart type</div>
+                    <div className="dc__text__subtitle fs-13 fw-6 cn-9 lh-16">{chartName}</div>
```
Comment (sohel-rp) [inline src/components/EnvironmentOverride/DeploymentTemplateOverride.tsx:308]: can we either use `dc__info-subtitle` class or change this to same pattern.

File: src/components/app/details/triggerView/EmptyStateCIMaterial.tsx [hunk 2]
```diff
@@ -28,39 +28,39 @@ export class EmptyStateCIMaterial extends Component<EmptyStateCIMaterialProps> {
     if (this.props.isRepoError) {
       return {
         img: <img src={ErrorImage} alt="no commits found" className="empty-state__img--ci-material" />,
-        title: <h1 className="empty__title">{this.props.repoErrorMsg}</h1>,
+        title: <h1 className="dc__empty__title">{this.props.repoErrorMsg}</h1>,
```
Comment (sohel-rp) [inline src/components/app/details/triggerView/EmptyStateCIMaterial.tsx:31]: Use the same pattern something like - `dc__empty-title`

File: src/components/app/list/list.css [hunk 3]
```diff
@@ -347,16 +347,16 @@
...
     color: var(--N700);
 }
 
-.app-list__row .app-commit__hash:hover {
+.app-list__row .dc__app-commit__hash:hover {
     color: var(--B500);
 }
 
-.app-commit__hash:hover .stroke-color {
+.dc__app-commit__hash:hover .stroke-color {
```
Comment (vivek-devtron) [inline src/components/app/list/list.css:359]: we should not directly overwrite global css. it should be wrapped in parent container

File: src/components/bulkEdits/bulkEdit.css [hunk 4]
```diff
@@ -36,22 +36,22 @@
...
 }
 
 .border-top {
-    border-top: 1px solid #d0d4d9;
+    border-top: 1px solid var(--N200);
 }
 
-.border-right {
-    border-right: 1px solid #d0d4d9;
+.dc__border-right {
```
Comment (vivek-devtron) [inline src/components/bulkEdits/bulkEdit.css:46]: this should be moved into global CSS.

File: src/components/bulkEdits/bulkEdit.css [hunk 5]
```diff
@@ -36,22 +36,22 @@
...
+    border-right: 1px solid var(--N200);
 }
 
 .bulk-output-drawer {
     height: 280px;
 }
 
-.tertiary-tab__radio input:checked ~ .tertiary-output-tab {
+.dc__tertiary-tab__radio input:checked ~ .tertiary-output-tab {
```
Comment (vivek-devtron) [inline src/components/bulkEdits/bulkEdit.css:54]: Global CSS should overwritten only in some wrapped container component

File: src/components/checkList/AppCheckList.tsx [hunk 6]
```diff
@@ -17,15 +17,15 @@ export class AppCheckList extends Component<AppCheckListProps, AppCheckListState
                 <p className="m-0 fs-14">To deploy custom application</p>
             </div>
             <div className="fs-13 mb-12">
-                {!hostUrl && <NavLink to={`${URLS.GLOBAL_CONFIG_HOST_URL}`} className="no-decor mt-8 flex left" style={{ ['color']: hostUrl ? `#767d84` : `#0066cc` }} >
```
Comment (vivek-devtron) [inline src/components/checkList/AppCheckList.tsx:20]: please change #0066cc to var(--B500) and #767d84 to var(--N500)

File: src/components/checkList/ChartCheckList.tsx [hunk 7]
```diff
@@ -17,9 +17,9 @@ export class ChartCheckList extends Component<ChartCheckListProps, {}> {
                     <div className="fs-14">To deploy chart</div>
                 </div>
                 <div className="fs-13">
-                    {!this.props.chartChecklist.project && <NavLink to={`${URLS.GLOBAL_CONFIG_PROJECT}`} className="no-decor  mt-8 flex left" style={{ ['color']: project ? `#767d84` : `#0066cc` }}>
+                    {!this.props.chartChecklist.project && <NavLink to={`${URLS.GLOBAL_CONFIG_PROJECT}`} className="dc__no-decor  mt-8 flex left" style={{ ['color']: project ? `#767d84` : `#0066cc` }}>
```
Comment (vivek-devtron) [inline src/components/checkList/ChartCheckList.tsx:20]: please change #0066cc to var(--B500) and #767d84 to var(--N500)

File: src/components/ciPipeline/CIPipelineAdvanced.tsx [hunk 8]
```diff
@@ -66,8 +66,8 @@ export class CIPipelineAdvanced extends Component<CIPipelineAdvancedProps, {}> {
                     this.props.addEmptyStage(key)
                 }}
             >
-                <Add className="icon-dim-24 fcb-5 vertical-align-middle mr-16" />
-                <span className="artifact__add">Add Stage</span>
+                <Add className="icon-dim-24 fcb-5 dc__vertical-align-middle mr-16" />
+                <span className="dc__artifact__add">Add Stage</span>
```
Comment (vivek-devtron) [inline src/components/ciPipeline/CIPipelineAdvanced.tsx:70]: please change dc__artifact__add to dc__artifact-add

File: src/components/ciPipeline/ciPipeline.css [hunk 9]
```diff
@@ -302,7 +298,7 @@
     border-bottom: 1px solid var(--N100);
 }
 
-.text-uppercase {
+.dc__uppercase {
```
Comment (vivek-devtron) [inline src/components/ciPipeline/ciPipeline.css:301]: please move this to base

File: src/components/common/DeprecatedUpdateWarn.tsx [hunk 10]
```diff
@@ -6,7 +6,7 @@ export class DeprecatedWarn extends Component {
         return (
             <div className="flex left">
                 <img src={triangleAlert} alt="" className="icon-dim-16"/>
-                <span className="deprecated-warn__text">Deprecated</span>
+                <span className="dc__deprecated-warn__text">Deprecated</span>
```
Comment (vivek-devtron) [inline src/components/common/DeprecatedUpdateWarn.tsx:9]: please change dc__deprecated-warn__text to dc__deprecated-warn-text

File: src/components/app/list/list.css (hunk 3 above)
Comment (shivani170) [reply src/components/app/list/list.css:367]: As discussed earlier its been using multiple places. Wrapping isn’t possible in that case

**Review by vivek-devtron — APPROVED** [review]

**Pull request merged**